native/common/include/jp_classhints.h

```diff
@@ -22,10 +22,14 @@ class JPConversion
 public:
 	virtual ~JPConversion();
 
+	// GCOVR_EXCL_START
```

Comment:
if the default is never called, why cant it be pure virtual then? Can't it be tested at all?

Reply:
It can be pure virtual, but we have a handful of implementations that are missing a match method because the logic for the match is done in the getJavaConversion method rather than a match (because resources are held privately).

There is no way to hit this method currently.

Reply:
I see